keps/sig-network/2030-topology-aware-proxying/README.md

```diff
@@ -116,40 +119,17 @@ This approach does not allow a single EndpointSlice to target multiple zones or
 regions. Any approach that enabled that would be significantly more complicated.
 The initial proposal in [KEP
 #2004](https://github.com/kubernetes/enhancements/issues/2004) suggests that
-this won't be an issue. 
+this won't be an issue.
 
 ## Design Details
 
-### API Consumption
-
-EndpointSlice API consumers can use the following selectors to consume
-EndpointSlices:
-
-#### Select EndpointSlices Without Labels
-This is required to support producers of EndpointSlices that don't set these
-labels, including older versions of the EndpointSlice controller.
-
-```
-matchExpressions:
-  - {key: endpointslice.kubernetes.io/for-zone, operator: DoesNotExist}
-  - {key: endpointslice.kubernetes.io/for-region, operator: DoesNotExist}
-```
-
-#### Select EndpointSlices With Matching Zone
-```
-matchLabels:
-  endpointslice.kubernetes.io/for-zone: example-zone
-```
-
-#### Select EndpointSlices With Matching Region
-```
-matchLabels:
-  endpointslice.kubernetes.io/for-region: example-region
-```
-
-#### Kube-Proxy
 When the `EndpointSliceSubsetting` feature gate is set to true on `Kube-Proxy`,
-it will use these selectors to filter EndpointSlices.
+Kube-Proxy will be updated to use the `endpointslice.kubernetes.io/for-zone` and
+`endpointslice.kubernetes.io/for-region` labels. When kube-proxy is able to
+detect the zone or region it is running in and there are EndpointSlices matching
+the zone or region for a Service, only those endpoints will be used. If not,
+existing behavior will be used to route traffic randomly to all endpoints for a
+Service.
```

Comment:
Can you add an explicit sentence that in all cases, all EndpointSlice objects will be watched and filtering will be done purely on kube-proxy side.

Reply:
Updated, thanks!